daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -42,8 +45,7 @@ def _suspend_device(self):
 
         Get the current brightness level, store it for later and then set the brightness to 0
         """
-        self.suspend_args.clear()
-        self.suspend_args['brightness'] = (_da_get_logo_brightness(self))
+        self.brightness = (_da_get_logo_brightness(self))
```

Comment:
I meant `self.suspend_args['brightness'] = _da_get_logo_brightness(self)` here and `logo_brightness = self.suspend_args.get('brightness', (100, 100))` in the resume method

Reply:
Ok, should be good this time

Reply:
shit sorry :D It should be `self.suspend_args.get('brightness', 100)` , otherwise the fallback when brightness isn't set is wrong :D Sorry I missed that yesterday

Reply:
No prob. Fixed.